tests/manage/mcg/conftest.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import OPENSHIFT_DEDICATED_PLATFORM
+
+log = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items):
+    """
+    A pytest hook to filter out mcg tests
+    when running on openshift dedicated platform
+
+    Args:
+        items: list of collected tests
+
+    """
+    if (
+        config.ENV_DATA["platform"].lower() == OPENSHIFT_DEDICATED_PLATFORM
+        and float(config.ENV_DATA["ocs_version"]) < 4.8
```

Comment:
will MCG be deployed with dedicated and OCS 4.8?

Reply:
I checked with engineering for the exact version, the exact version is not yet decided. I am removing the condition from the code with TODO comment